src/main/kotlin/no/nav/familie/ks/sak/kjerne/beregning/TilkjentYtelseValidator.kt

```diff
@@ -34,4 +44,29 @@ object TilkjentYtelseValidator {
     private fun hentBarnasAndeler(andeler: List<AndelTilkjentYtelse>, barna: List<Person>) = barna.map { barn ->
         barn to andeler.filter { it.aktør == barn.aktør }
     }
+
+    private fun validerAtBeløpForPartStemmerMedSatser(person: Person, andeler: List<AndelTilkjentYtelse>) {
+        val maksAntallAndeler = if (person.type == PersonType.BARN) 1 else 2
```

Comment:
`maksAntallAndeler` skal være 0 for søker, men vet ikke om vi trenger å sjekke det her i det hele tatt 🤔

Reply:
fikset og kastet en feil hvis søkers andeler ikke er tom